addons/account/tests/test_unexpected_invoice.py

```diff
@@ -33,12 +33,25 @@ def test_higher_amount(self):
         base = self.env['account.move'].create([self._invoice_vals(price_unit=100) for i in range(10)])
         base.action_post()
 
-        move = self.env['account.move'].create(self._invoice_vals(price_unit=100))
-        self.assertFalse(move.abnormal_amount_warning)
-        move = self.env['account.move'].create(self._invoice_vals(price_unit=200))
-        self.assertTrue(move.abnormal_amount_warning)
-        move = self.env['account.move'].create(self._invoice_vals(price_unit=50))
-        self.assertTrue(move.abnormal_amount_warning)
+        bill_1, bill_2, bill_3 = self.env['account.move'].create([
+            self._invoice_vals(price_unit=100),
+            self._invoice_vals(price_unit=200),
+            self._invoice_vals(price_unit=50),
+        ])
+        self.assertFalse(bill_1.abnormal_amount_warning)
+        self.assertTrue(bill_2.abnormal_amount_warning)
+        self.assertTrue(bill_3.abnormal_amount_warning)
+
+        self.env = self.env(context={
+            k: v for k, v in self.env.context.items() if k != 'disable_abnormal_invoice_detection'
+        })
```

Comment:
Also, this is changing a class property shared with the other tests of this class

Reply:
Yes, I haven't finished.
I figured out before pushing because the test is still broken.
I have to finish it tomorrow.
My plan is to save the env and restore it at the end. 

Now that I'm thinking about it, I'm not sure it writes on the class itself rather than creating an attribute on the instance that shadows the class attribute.

I have to check that too.